TESTS/mbedtls/ecp/main.cpp

```diff
@@ -20,8 +20,6 @@
 #include "utest.h"
 #include "rtos.h"
 
-using namespace utest::v1;
-
```

Comment:
I'm fine with your approach, I only add a statement that future functions\types should also include the `utest::v1` prefix.